arches/isa_json/gen_uarch_rv64v_json.py

```diff
@@ -217,24 +217,165 @@
+
+# Vector Floating-Point Instructions: Vector Widening Floating-Point Add/Subtract Instructions
+    "vfwadd.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfwadd.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfwsub.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfwsub.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfwadd.wv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfwadd.wf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfwsub.wv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfwsub.wf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+# Vector Floating-Point Instructions: Vector Single-Width Floating-Point Multiply/Divide Instructions
+    "vfmul.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfmul.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfdiv.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 25},
+    "vfdiv.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 25},
+
+    "vfrdiv.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" :25},
+
+# Vector Floating-Point Instructions: Vector Widening Floating-Point Multiply
+    "vfwmul.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 3},
+    "vfwmul.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 3},
+
+# Vector Floating-Point Instructions: Vector Single-Width Floating-Point Fused Multiply-Add Instructions
+    "vfmacc.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfmacc.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfnmacc.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfnmacc.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfmsac.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfmsac.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfnmsac.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfnmsac.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfmadd.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfmadd.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfnmadd.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfnmadd.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfmsub.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfmsub.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+    "vfnmsub.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+    "vfnmsub.vf" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
+
+# Vector Floating-Point Instructions: Vector Widening Floating-Point Fused Multiply-Add Instructions
+    "vfwmacc.vv" :  {"pipe" : "vfloat", "uop_gen" : "ARITH", "latency" : 6},
```

Comment:
These should use the `ARITH_MAC` which adds the destination of each uop as a source.